core/src/commands/command_helper.cpp

```diff
@@ -84,26 +84,31 @@ int AOClient::genRand(int min, int max)
 #endif
 }
 
-void AOClient::diceThrower(int argc, QStringList argv, bool p_roll)
+void AOClient::diceThrower(int sides, int dice, bool p_roll, int roll_modifier)
 {
-    int l_sides = 6;
-    int l_dice = 1;
+    if (sides < 0 || dice < 0 || sides > 99999 || dice > 99999) {
```

Comment:
```suggestion
    if (sides < 0 || dice < 0 || sides > ConfigManager::diceMaxValue() || dice > ConfigManager::diceMaxDice()) {
```
The lower bound is fine, but the upper bound is defined in config.ini